src/MudBlazor/TScripts/mudInput.js

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) MudBlazor 2021
+// MudBlazor licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+class MudInput {
+    resetValue(id) {
+        let input = document.getElementById(id);
```

Comment:
tiny nitpick, always use const:

```suggestion
        const input = document.getElementById(id);
```